ff/serving/provider/offline.go

```diff
@@ -157,7 +173,7 @@ func (store *memoryOfflineStore) AsOfflineStore() (OfflineStore, error) {
 	return store, nil
 }
 
-func (store *memoryOfflineStore) CreateResourceTable(id ResourceID) (OfflineTable, error) {
+func (store *memoryOfflineStore) CreateResourceTable(id ResourceID, schema SerializedSchema) (OfflineTable, error) {
```

Comment:
Can I get some feedback on if this is an okay way to deal with the schema? (Passing it to CreateResourceTable)

Reply:
by schema you mean SQL schema?

Reply:
Table schema, will change name to clarify